frontend/app.tsx

```diff
@@ -197,9 +117,16 @@ function issueCountView(
 function filteredIssuesView(
   source: IMemorableSource<Issue, Map<string, Issue>>,
   order: Order,
-  filter: (i: Issue) => boolean
+  filters: (((i: Issue) => boolean) | null)[]
 ) {
-  return source.stream.filter(filter).materialize((l: Issue, r: Issue) => {
+  let { stream } = source;
+  for (const f of filters) {
+    if (!f) {
+      continue;
+    }
+    stream = stream.filter(f);
+  }
```

Comment:
filters are just functions and we append each to the stream. Pretty simple from a processing perspecteive.